rellic/AST/Z3ConvVisitor.cpp

```diff
@@ -202,25 +202,28 @@ clang::Expr *Z3ConvVisitor::CreateLiteralExpr(z3::expr z_expr) {
     } break;
 
     case Z3_BV_SORT: {
-      auto s_size{GetZ3SortSize(sort)};
-      auto &ti{ast_ctx->getTargetInfo()};
-      auto t_size{ti.getTypeWidth(ti.getLeastIntTypeByWidth(s_size, 0))};
-      auto t_lit{ast_ctx->getIntTypeForBitwidth(t_size, 0)};
-      llvm::APInt val(t_size, Z3_get_numeral_string(z_expr.ctx(), z_expr), 10);
-      switch (clang::cast<clang::BuiltinType>(t_lit)->getKind()) {
+      auto type{GetLeastIntTypeForBitWidth(*ast_ctx, GetZ3SortSize(sort),
+                                           /*sign=*/0)};
+      auto size{ast_ctx->getTypeSize(type)};
+      llvm::APInt val(size, Z3_get_numeral_string(z_expr.ctx(), z_expr), 10);
+      switch (clang::cast<clang::BuiltinType>(type)->getKind()) {
```

Comment:
Can you add a comment here about why `ushort` and `uchar` need special handling, but others do not?